javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,73 @@
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content, id }); 
+  return `Note with id ${id} saved successfully!`;
+}
+
+console.log(saveNote('going on a run', 1)); 
+console.log(saveNote('meeting with friends', 2)); 
+
+// Get a note by id
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++) {
```

Comment:
Good work with this one as well! 😄 This file contains a lot of for loops where numbers are being fiddled with. I would recommend using `for...of` syntax especially when you do not actually need the index of the element. 🙂